metabase/metabase
# [FIX: #39152] Public links refer to `null` when question is new

Closes #39152

### Description

When questions are new, they don't have an ID and aren't able to possess a public_uuid since they aren't saved. To solve this, we need to make sure that the user has saved the question before they try to create a public link for it.

### How to verify

1. Create a new question
2. See that, when trying to click the "Share" button in the bottom right corner, that a `First, save your question` modal pops up first.

### Demo

_Upload a demo video or before/after screenshots if sensible or remove the section_

### Checklist
